src/components/ActionButtons.jsx

```diff
@@ -0,0 +1,57 @@
+/* global _ */
+import React from 'react';
+import classnames from 'classnames';
+
+export default class ActionButtons extends React.Component {
+  constructor(props) {
+    super(props);
+    this.elem = props.elem;
+    this.poi = props.poi;
+    this.favoriteClick = this.favoriteClick.bind(this);
```

Comment:
There is a simpler alternative to that : just declare `favoriteClick` as an arrow function.
And now your constructor is empty, so you can remove it :)